config/routes.rb

```diff
@@ -12,4 +12,12 @@
       end
     end
   end
+
+  get '/:token/confirm_email/', :to => "api/v1/users#confirm_email", as: 'confirm_email'
+
+  namespace 'api' do
+    namespace 'v1' do
+      resources :users, only: [:confirm_email]
```

Comment:
лучше не использовать такие вот кастомные экшены и засорять ими контроллер. С годами, применяя такой подход, твой контроллер будет 2-3к строк и все будут плеваться от него
предлагаю вот что
```suggestion
      namespace :users do
        resources :confirmations, only: [:new]
      end
```

Reply:
И экшен соответственно